core/services/ocrcommon/telemetry.go

```diff
@@ -295,7 +297,7 @@ func (e *EnhancedTelemetryService[T]) collectMercuryEnhancedTelemetry(d Enhanced
 	var bt uint64
 	// v1+v2+v3 fields
 	bp := big.NewInt(0)
-	//v1+v3 fields
+	// v1+v3 fields
```

Comment:
do we want to add v4 telemetry?

Reply:
Good catch. I've added a new enum and field for the market status. @mjk90 do you mind also taking a look at the `EnhancedEAMercury` proto and see if it makes sense?